packages/block-library/src/post-tags/edit.js

```diff
@@ -1,42 +1,77 @@
 /**
  * WordPress dependencies
  */
-import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { useEntityProp } from '@wordpress/core-data';
+import { Warning, __experimentalBlock as Block } from '@wordpress/block-editor';
 import { useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
-function PostTagsDisplay() {
-	const [ tags ] = useEntityProp( 'postType', 'post', 'tags' );
+export default function PostTagsEdit( { context } ) {
+	const [ tags ] = useEntityProp(
+		'postType',
+		context.postType,
+		'tags',
+		context.postId
+	);
 	const tagLinks = useSelect(
 		( select ) => {
 			const { getEntityRecord } = select( 'core' );
 			let loaded = true;
-			const links = tags.map( ( tagId ) => {
-				const tag = getEntityRecord( 'taxonomy', 'post_tag', tagId );
-				if ( ! tag ) {
-					return ( loaded = false );
-				}
-				return (
-					<a key={ tagId } href={ tag.link }>
-						{ tag.name }
-					</a>
-				);
-			} );
+			let links;
+
+			if ( tags ) {
+				links = tags.map( ( tagId ) => {
+					const tag = getEntityRecord(
+						'taxonomy',
+						'post_tag',
+						tagId
+					);
+					if ( ! tag ) {
+						return ( loaded = false );
+					}
+					return (
+						<a key={ tagId } href={ tag.link }>
+							{ tag.name }
+						</a>
+					);
+				} );
+			}
+
 			return loaded && links;
```

Comment:
I notice this was not added or changed here, but this seems a bit odd.  So if we have a problem finding 1 of the tags in the list, we return no tagLinks at all?  
It seems like this may need to be updated in the future so that it returns all the tags that are successfully found 🤔

Reply:
That's a good point 🤔 Since it seems out of scope for this PR, would you feel comfortable if we rolled that UX discussion into https://github.com/WordPress/gutenberg/pull/24069?